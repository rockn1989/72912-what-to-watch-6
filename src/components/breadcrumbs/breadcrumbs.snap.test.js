import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import BreadCrumbs from './breadcrumbs';

it(`Should Breadcrumbs render correctly`, () => {
  const history = createMemoryHistory();
  const data = {id: 1, name: `film`};
  const {container} = render(
      <Router history={history}>
        <BreadCrumbs {...data} />
      </Router>
  );
  expect(container).toMatchSnapshot();
});
