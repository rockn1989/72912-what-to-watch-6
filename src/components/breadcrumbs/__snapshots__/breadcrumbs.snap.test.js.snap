// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Breadcrumbs render correctly 1`] = `
<div>
  <nav
    class="breadcrumbs"
  >
    <ul
      class="breadcrumbs__list"
    >
      <li
        class="breadcrumbs__item"
      >
        <a
          class="breadcrumbs__link"
          href="/films/1"
        >
          film
        </a>
      </li>
      <li
        class="breadcrumbs__item"
      >
        <a
          class="breadcrumbs__link"
        >
          Add review
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
